harness/determined/common/experimental/determined.py

```diff
@@ -121,6 +121,9 @@ def create_experiment(
         config: Union[str, pathlib.Path, Dict],
         model_dir: Optional[Union[str, pathlib.Path]] = None,
         includes: Optional[Iterable[Union[str, pathlib.Path]]] = None,
+        parent_id: Optional[int] = None,
```

Comment:
Please don't forget to update the docstring.

Also, while you're at it, could you please fix the indenting on the `includes` line?

Reply:
Stylistically, what I'm looking for here looks like the following (though I didn't count the characters, so the exact strings might be different)
```
        Arguments:
            config(string, pathlib.Path, dictionary): experiment config filename (.yaml)
                or a dict.
            model_dir(string, optional): directory containing model definition. (default: ``None``)
            includes(Iterable[Union[str, pathlib.Path]], optional): Additional files or
                directories to include in the model definition. (default: ``None``)
            parent_id(int, optional): The id of the parent experiment. (default: ``None``)
            project_id(int, optional): The id of the project this experiment should belong to.
                (default: ``None``)
            template(string, optional): The name of the template for the experiment.
                See :ref:`config-template` for moredetails. (default: ``None``)
```
or
```
        Arguments:
            config: experiment config filename (.yaml) or a dict.
            model_dir: directory containing model definition.
            includes: Additional files or directories to include in the model definition.
            parent_id: The id of the parent experiment.
            project_id: The id of the project this experiment should belong to.
            template: The name of the template for the experiment. See :ref:`config-template` for
                more details.
```

This follows our style guide https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

(this style guides says that when the function has a type annotation (like this one does) the types are optional. I think the same should be true for the default, unless the default needs more explanation than what shows up in the generated docs.)

Reply:
Thanks!